packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,174 @@
+import React, {
+  Children,
+  createRef,
+  isValidElement,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react"
+import styled from "styled-components"
+import { useCursor } from "use-cursor"
+import { color, space } from "../../helpers"
+import { ChevronIcon } from "../../svgs"
+import { Box, BoxProps } from "../Box"
+import { Clickable } from "../Clickable"
+import { Skip } from "../Skip"
+import { VisuallyHidden } from "../VisuallyHidden"
+import { paginate } from "./utils"
+
+const Container = styled(Box)`
+  position: relative;
+  width: 100%;
+`
+
+const Viewport = styled(Box)`
+  width: 100%;
+  overflow: hidden;
+`
+
+const Rail = styled(Box)`
+  display: flex;
+  height: 100%;
+  transition: transform 500ms;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+`
+
+const ARROW_WIDTH = space(4)
```

Comment:
The reason this is a hardcoded value is because this is the available margin in our page wrapper. The arrows sit outside of the width of the carousel, which is a little bit weird but makes sense in our universe.